config/templates/osd-metrics-exporter-csv-template.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: osd-metrics-exporter
```

Comment:
I had a look for reference to other operators, and convention is to have {operator-name}-{version} for the name field (like [in aws-account-operator](https://github.com/openshift/aws-account-operator/blob/1671948c909525c5df58e81464755a70b452d426/config/templates/aws-account-operator-csv-template.yaml#L4) )